src/wallet/rpcwallet.cpp

```diff
@@ -2654,6 +2654,11 @@ UniValue fundrawtransaction(const UniValue& params, bool fHelp)
 
 UniValue regeneratemintpool(const UniValue &params, bool fHelp) {
 
+        if (fHelp || params.size() > 0)
+	        throw runtime_error(
+       	        	"regeneratemintpool\n"
+       		        "\nIf issues exist with the keys that map to mintpool entries in the DB, this function corrects them.\n"
+                );
```

Comment:
We should provides examples too so it will be the same as other Bitcoin RPCs.

Reply:
I had considered that but no arguments are being passed. What do you suggest in this case?

Reply:
We should show it without arguments like `getrawchangeaddress`.

Reply:
Updated to include this.